<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
<<<<<<< Updated upstream
    <width>1097</width>
    <height>707</height>
=======
    <width>1131</width>
    <height>741</height>
>>>>>>> Stashed changes
   </rect>
  </property>
  <property name="windowTitle">
   <string>Math Scriptum</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ShapeArea" name="Shapearea" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>451</width>
<<<<<<< Updated upstream
      <height>591</height>
=======
      <height>341</height>
>>>>>>> Stashed changes
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="NoteArea" native="true">
    <property name="geometry">
     <rect>
<<<<<<< Updated upstream
      <x>600</x>
      <y>50</y>
      <width>611</width>
      <height>251</height>
=======
      <x>450</x>
      <y>60</y>
      <width>611</width>
      <height>621</height>
>>>>>>> Stashed changes
     </rect>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
<<<<<<< Updated upstream
       <y>10</y>
       <width>551</width>
       <height>231</height>
=======
       <y>0</y>
       <width>611</width>
       <height>621</height>
>>>>>>> Stashed changes
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="EquationsArea" name="equationsArea" native="true">
    <property name="geometry">
     <rect>
<<<<<<< Updated upstream
      <x>590</x>
      <y>360</y>
      <width>501</width>
=======
      <x>0</x>
      <y>400</y>
      <width>451</width>
>>>>>>> Stashed changes
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
<<<<<<< Updated upstream
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>326</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sScale"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sInterval"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Scale/Interval</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sCount"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>106</width>
      <height>540</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="CycloidButton">
       <property name="text">
        <string>Cycloid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="astroid_Button">
       <property name="text">
        <string>Astroid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="circle_Button">
       <property name="text">
        <string>circle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ellipse_Button">
       <property name="text">
        <string>Ellipse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hexagon_Button">
       <property name="text">
        <string>hexagon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hypo_Button">
       <property name="text">
        <string>Hypo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="invertedCircle_Button">
       <property name="text">
        <string>inverted circle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="huyg_Button">
       <property name="text">
        <string>huygens</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="line_Button">
       <property name="text">
        <string>Lines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createSaveFileButton">
       <property name="text">
        <string>Create Save File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectSaveFileButton">
       <property name="text">
        <string>Select Save File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pasteFromFile">
       <property name="text">
        <string>Paste From File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveProgress">
       <property name="text">
        <string>Save Progress</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveImage">
       <property name="text">
        <string>Save Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="limitsButton">
       <property name="text">
        <string>Limits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="derivativesButton">
       <property name="text">
        <string>Derivatives</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="integralsButton">
       <property name="text">
        <string>Integrals</string>
       </property>
      </widget>
=======
    <layout class="QFormLayout" name="formLayout"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Bodoni MT</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>true</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Size</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="sCount">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Bodoni MT</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>true</italic>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QDoubleSpinBox" name="sInterval">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Bodoni MT</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>true</italic>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="4" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Bodoni MT</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>true</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Stepper</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QDoubleSpinBox" name="sScale">
           <property name="font">
            <font>
             <family>Bodoni MT</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>true</italic>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QPushButton" name="openButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Open</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="printButton_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Print to PDF</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Bodoni MT</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>true</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Parametric</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
>>>>>>> Stashed changes
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
<<<<<<< Updated upstream
      <x>570</x>
      <y>320</y>
      <width>507</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>158</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="xRecipButton">
       <property name="text">
        <string>1/x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sqrtButton">
       <property name="text">
        <string>Square Root</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="parabolaButton">
       <property name="text">
        <string>Parabola</string>
=======
      <x>640</x>
      <y>10</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="createSaveState">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Create Save State</string>
>>>>>>> Stashed changes
       </property>
      </widget>
     </item>
     <item>
<<<<<<< Updated upstream
      <widget class="QPushButton" name="lineButton">
       <property name="text">
        <string>line</string>
=======
      <widget class="QPushButton" name="selectSaveState">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Select Save State</string>
>>>>>>> Stashed changes
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
<<<<<<< Updated upstream
     <width>1097</width>
     <height>23</height>
=======
     <width>1131</width>
     <height>26</height>
>>>>>>> Stashed changes
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShapeArea</class>
   <extends>QWidget</extends>
   <header location="global">shapearea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EquationsArea</class>
   <extends>QWidget</extends>
   <header location="global">equationsarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
